processes/consumer/configs.go

```diff
@@ -44,7 +48,25 @@ func CommitOffset(ctx context.Context, topic string, partitionsToOffset map[stri
 	for _, msgs := range partitionsToOffset {
 		for _, msg := range msgs {
 			if msg.KafkaMsg != nil {
-				err = topicToConsumer.Get(topic).CommitMessages(ctx, *msg.KafkaMsg)
+				for attempts := 0; attempts < 5; attempts++ {
+					err = topicToConsumer.Get(topic).CommitMessages(ctx, *msg.KafkaMsg)
+					if err == nil {
+						break
+					}
+
+					if kafkalib.IsRetryableErr(err) {
+						jitterDuration := jitter.JitterMs(1500, attempts)
+						logger.FromContext(ctx).WithError(err).WithFields(map[string]interface{}{
+							"attempts":            attempts,
+							"sleep duration (ms)": jitterDuration,
+						}).Info("encountered a retryable error, will sleep and retry...")
+
+						time.Sleep(time.Duration(jitterDuration) * time.Millisecond)
+					} else if err != nil {
```

Comment:
`err` will never be `nil` here because of the check on line 53.

Reply:
Yeah, I can make it `else` instead of `else if`